cpp/src/KIM_ModelImplementation.cpp

```diff
@@ -3005,23 +3060,32 @@ int ModelImplementation::InitializeParameterizedModel(
 #endif
 
   // write parameter files to scratch space
-  error = WriteParameterFiles();
-  if (error)
+  error
```

Comment:
@ellio167  identifier for error is undefined here. It is defined within the `if` condition above. I think minor correction should be done.
 
```cpp
int error;
#if ERROR_VERBOSITY
  error = (!requestedLengthUnit.Known()) || (!requestedEnergyUnit.Known())
              || (!requestedChargeUnit.Known())
              || (!requestedTemperatureUnit.Known())
              || (!requestedTimeUnit.Known());
  if (error)
  {
    LOG_ERROR("Invalid arguments.");
    LOG_DEBUG("Exit 1=" + callString);
    return true;
  }
#endif
```

Reply:
good; fixed.  Thanks.